ui/src/main/webapp/src/app/project/migration-project-form.component.ts

```diff
@@ -55,6 +56,9 @@ export class MigrationProjectFormComponent extends FormComponent implements OnIn
             }
 
             this.isInWizard = flatRouteData.data.hasOwnProperty('wizard') && flatRouteData.data['wizard'];
+
+            if(!this.editMode) // Creating a new project.
+                this._migrationProjectService.deleteProvisionalProjects().subscribe(res => {});
```

Comment:
Here is the commit, if you are curious: https://github.com/windup/windup-web/commit/35e3529301badbc67f0fbf273af3d95944e554e5

forkJoin would be a better solution really.